examples/chip-tool/entitlements/codesign.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env -S python3 -B
```

Comment:
And ideally only one copy of the codesign script too, somewhere under `scripts`.  Follow-up for that?

Reply:
Followup